### DISCUSSION TITLE
今後のスケジュール

### DISCUSSION THREAD

**Discussion by shimizu-1111** [22w2076e/SoftwareEcosystem-Group2#4] [General]
スケジュール感を共有しましょう

**Comment by shimizu-1111** [comment]
1/11の講義で進捗報告しましょう

**Comment by shimizu-1111** [comment]
今年中にデータの整形をやる →@shimizu-1111 
リポジトリ合算版と，リポジトリごとにまとめるコードを作成する．

条件
・リポジトリは上位100件を取得する．
・ユーザは貢献度順に上位100件を取得する．

注意点
apiのアクセス制限が厳しいのでトークンを発行して緩和する．

**Reply by shimizu-1111** [reply]
すみません．
コード作成が年明けになりそうです．
3日までに作成します．

**Reply by ayame113** [reply]
時間があったので、自分でできる範囲のところまでやってみました。https://github.com/22w2076e/SoftwareEcosystem-Group2/pull/5

~~いまこのリポジトリにファイルを追加する権限が私には無いみたいなので、私のフォークからデータが見られるようにしてあります。~~

- ~~https://github.com/ayame113/SoftwareEcosystem-Group2/tree/create-dataset~~
- ~~https://github.com/ayame113/SoftwareEcosystem-Group2/blob/create-dataset/4_result.json~~
- ~~https://github.com/ayame113/SoftwareEcosystem-Group2/blob/create-dataset/4_contributions.json~~

<br>

- https://github.com/22w2076e/SoftwareEcosystem-Group2
- ユーザーのデータ（全リポジトリ合計）：https://github.com/22w2076e/SoftwareEcosystem-Group2/blob/main/4_result.json
- ユーザーの貢献数（リポジトリごと）：https://github.com/22w2076e/SoftwareEcosystem-Group2/blob/main/4_contributions.json

**Reply by 22w2076e** [reply]
collaboratorに追加しておきました

**Reply by ayame113** [reply]
ありがとうございます！

**Reply by shimizu-1111** [reply]
ありがとうございます．
先ほど内容を確認しました．
お任せしてしまい申し訳ありません．

**Comment by ayame113** [comment]
> グループ演習（1/11）
引き続きグループ演習に取り組んでください．授業は無いので遠隔・非同期で取り組んでください．いつもの時間に教室で実施してもらっても大丈夫です．
https://lms.ealps.shinshu-u.ac.jp/2022/st/course/view.php?id=1920

とりあえず教室に集まりましょうか？

**Reply by recofita** [reply]
分かりました。

**Comment by shimizu-1111** [comment]
来週の授業前までにグラフ作成等のデータ可視化をしておく
できればスライドにグラフを追加してまとめておく

来週の授業ではスライドをまとめる

**Comment by shimizu-1111** [comment]
すみません．

体調がすぐれないため今日の授業はオンラインで参加させていただきたいです．
差し支えなければこちらのmeetに参加してくださいますようお願いします．
URL：https://meet.google.com/wra-zhcc-fyz

なにか問題がございましたら，こちらのreply欄に書き込みをお願いします．

**Comment by ayame113** [comment]
次回は1月23日（月）の10:00に集まりましょう。
（忙しい場合は欠席してもOKです）

303号室（自分の研究室）に集まりましょう。
